README.md

```diff
@@ -199,5 +206,7 @@
 * [Subtractive synths explained](https://www.residentadvisor.net/features/1351)
 
 ## Experiments
-
 * [generative.fm](https://generative.fm): Endlessly unique ambient music
+
+## Practice Tools
```

Comment:
I think we should move this to an existing section. Post people seeing this might not know what "Practice Tools" is.